lib/sapience/sinatra.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require "sapience"
+require "sapience/extensions/sinatra/timings"
+require "sapience/extensions/sinatra/middleware/logging"
+
+module Sinatra
+  class Base
+    extend Sapience::Descendants
+  end
+end
+
+module Sapience
+  class Siantra
+    Sapience.configure
```

Comment:
typo :)  Siantra --> Sinatra

Reply:
Thanks for CR @jelamo , happy to see that you are missing Ruby :)

Reply:
oh was that Ruby? then leave the type what need fixing is the language :D

Reply:
That is a good one :D